contrib/build_sdk.sh

```diff
@@ -656,6 +779,7 @@ freeimage_pkg() {
         # manually copy header and library
         cp $freeimage_dir/Dist/FreeImage.h $install_dir/include || exit 1
         cp $freeimage_dir/Dist/libfreeimage* $install_dir/lib || exit 1
+        echo $? > $status_dir/$name.success
```

Comment:
this is actually equal to `echo 0 > $status_dir/$name.success`, but that's ok

Reply:
cool, left as is